odpc.client/src/features/publicatie/PublicatieDetails.vue

```diff
@@ -1,94 +1,77 @@
 <template>
-  <simple-spinner v-if="isFetching"></simple-spinner>
-
-  <form v-else aria-live="polite" @submit.prevent="submit">
-    <alert-inline v-if="error"
-      >Er is iets misgegaan bij het ophalen van de publicatie...</alert-inline
-    >
-
-    <fieldset v-else-if="publicatie">
-      <div class="form-group">
-        <label for="titel">Titel</label>
-
-        <input
-          id="titel"
-          type="text"
-          v-model="publicatie.officieleTitel"
-          required
-          aria-required="true"
-        />
-      </div>
-
-      <div class="form-group">
-        <label for="verkorte_titel">Verkorte titel</label>
-
-        <input id="verkorte_titel" type="text" v-model="publicatie.verkorteTitel" class="small" />
-      </div>
-
-      <div class="form-group">
-        <label for="omschrijving">Omschrijving</label>
-
-        <textarea id="omschrijving" v-model="publicatie.omschrijving" rows="4"></textarea>
-      </div>
-    </fieldset>
-
-    <div class="form-submit" :class="{ error }">
-      <router-link :to="{ name: 'publicaties' }" class="button button-secondary">{{
-        error ? "&lt; Terug" : "Annuleren"
-      }}</router-link>
-
-      <button v-if="publicatie && !error" type="submit" title="Opslaan">Opslaan</button>
+  <simple-spinner v-show="loading"></simple-spinner>
+
+  <form v-show="!loading" aria-live="polite" @submit.prevent="submit">
+    <section>
+      <publicatie-form
```

Comment:
Ik vind het 'exposen' van functies op een component niet zo'n mooi pattern. Is het een idee om de twee componenten alleen een model mee te geven (dus `publicatie-form` een `Publicatie` en `document-form` een `PublicatieDocument`), en het ophalen / opslaan in deze overkoepelende component te regelen? Misschien even overleggen?